tasks/src/tabular-regression/data.ts

```diff
@@ -0,0 +1,59 @@
+import type { TaskData } from "../Types";
+
+import { PIPELINE_DATA } from "../../../js/src/lib/interfaces/Types";
+import { TASKS_MODEL_LIBRARIES } from "../const";
+
+const taskData: TaskData = {
+	datasets: [
+		{
+			description: "Dataset containing tip amounts for food servers.",
+			id:          "scikit-learn/tips",
+		},
+		{
+			description: "Dataset containing mile-per-gallon (mpg) information for various vehicles",
+			id:          "scikit-learn/auto-mpg",
+		},
+		{
+			description: "Dataset containing measurements of fish at a market.",
+			id:          "brendenc/fish",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				table: [
+					["Mother's Height (cm)", "Father's Height (cm)"],
+					["162", "172",],
+					["165", "178"],
+					["170", "165"]
+
+				],
+				type: "tabular",
+			},
+		],
+		outputs: [
+			{
+				table: [["Baby's Height (cm)"], ["47"], ["50"], ["51"]],
```

Comment:
Looks good to me, but I feel like it would be good to put a business case, e.g. a small subset of auto-mpg dataset's columns 

<img width="768" alt="Ekran Resmi 2022-10-10 20 37 59" src="https://user-images.githubusercontent.com/53175384/194932032-45efa7b5-2952-4a34-a925-e868b2838f7e.png">

Reply:
Sounds good!

Reply:
Updated using auto-mpg, let me know if you want to include more features.